core/src/main/kotlin/DokkaGenerator.kt

```diff
@@ -31,7 +31,7 @@ class DokkaGenerator(
         val context = DokkaContext.create(configuration.pluginsClasspath, logger, platforms)
 
         logger.debug("Creating documentation models")
-        val modulesFromPlatforms = platforms.map { (pdata, _) -> translateDescriptors(pdata, context) }
+        val modulesFromPlatforms = platforms.map { (pdata, _) -> translateDescriptors(configuration.passesConfigurations.first().moduleName, pdata, context) }
```

Comment:
That is wrong. 

A bit earlier we are doing 
`val platforms: Map<PlatformData, EnvironmentAndFacade> = configuration.passesConfigurations.map {
            PlatformData(it.analysisPlatform, it.targets) to createEnvironmentAndFacade(it)
        }.toMap()`

so now we are actually iterating over a list of passesConfigurations after mapping. I would rather suggest adding `moduleName` to `PlatformData` class, so this lines would look as follows:

`val platforms: Map<PlatformData, EnvironmentAndFacade> = configuration.passesConfigurations.map {
            PlatformData(it.analysisPlatform, it.targets, it.moduleName) to createEnvironmentAndFacade(it)
        }.toMap()`

and then

`val modulesFromPlatforms = platforms.map { (pdata, _) -> translateDescriptors(pdata.moduleName, pdata, context) }`
You can even skip passing `moduleName` here, you can simply retrieve it from pdata later, inside `translateDescriptors` function.

Reply:
Changed.